conference.js

```diff
@@ -1247,11 +1247,28 @@ export default {
         }
 
         options.applicationName = interfaceConfig.APP_NAME;
-        options.getWiFiStatsMethod = getJitsiMeetGlobalNS().getWiFiStats;
+        options.getWiFiStatsMethod = this._getWiFiStatsMethod;
 
         return options;
     },
 
+    /**
+     * Returns the result of getWiFiStats from the global NS or does nothing
+     * (returns empty result).
+     * Fixes a concurrency problem where we need to pass a function when creating
+     * JitsiConference, but that method is added to the context later.
+     *
+     * @returns {Promise}
+     * @private
+     */
+    _getWiFiStatsMethod() {
+        const gloabalNS = getJitsiMeetGlobalNS();
+
+        return gloabalNS.getWiFiStats
+            ? gloabalNS.getWiFiStats()
+            : Promise.resolve('{}');
```

Comment:
I think this can fit on a single line, isn't it?

There was an issue with the returns like this one that the function will just return `gloabalNS.getWiFiStats` and ignore the rest. But maybe webpack and the optimiser fixes this anyway.

Reply:
Yep, just force push it.